frontend/src/services/apiService.ts

```diff
@@ -45,6 +45,32 @@ export const api = {
             );
         }
     },
+    events: {
+        list: ({ associationId }) => 
```

Comment:
Might not be up to date with the new API system, merge master into this branch to check

Reply:
The branch is up to date with master